### PR TITLE
Added almost every plugin in existence that is confirmed to be still working, but was missing from the list.

### PR DESCRIPTION
**Pull request by Cat-Lady**
Added almost every plugin in existence that is confirmed to be still working, but was missing from the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,14 +70,20 @@ Since this list strives for completeness rather than listing only the best, it's
...
 - [swizzle-player](https://github.com/Cat-Lady/swizzle-player/releases) - Allows to change player's swizzle in "Endless Sky" - and keep the colour after ES get updates that change goverments.txt data file.
 - [The Adamas Project](https://github.com/toilethinges/Adamas-Project) - Several new ships, many outfits, and a variety of other things to be discovered.
+- [The Enclave](https://github.com/Makuta-Miras/The-Enclave) - The Enclave focuses around the discovery of an isolated group of humans in a distant galaxy. Separated from the rest of their species for over half a millennium, how have these people developed?
+- [Ursa Polaris](https://steamcommunity.com/sharedfiles/filedetails/?id=631160921) - This plugin contains missions for your pilot to create a centrally located outfitter space station that stocks outfits from across the galaxy. It happens after main campaing and is aiming to be NOT overpowered.
```
Comment (MCOfficer) [inline README.md]: Is it worth linking the original when it's part of [es-plugin-maintenance](https://github.com/comnom/es-plugin-maintenance)?

Comment (Cat-Lady) [reply README.md]: Yes, cause the es-plugin-maintentance is - quite counter-intuitively to the name - currently unmaintained, and (for example) Ursa Polaris have much more recent version mainstream.

If anything, I would kick out the es-plugin-maintentance from list, or at least add a warning that it is not guaranteed to actually have recent versions of the plugins.

Comment (MCOfficer) [reply README.md]: cc @comnom

Comment (comnom) [reply README.md]: Probably in your best interest to take it off. It was made as a service to users, and doesnt account for authors returning. Ive also seen a new version of animated stars forked from the original broken one, so the version people are likely to use doesnt include my fix anyhow. A better solution would be for me to take stuff like that out, but unless someone else wants to take over that repo (ha), that aint gonna happen.

**Pull request merged**